### PR TITLE
Created toolbar and headline section-mobile version 

### PR DESCRIPTION
**Pull request by Zainab-Alzubair**
In this PR:
Added linters
Created HTML code for mobile toolbar and headline section
Styled the toggled toolbar and the headline section 
Added the social media icons 
Added the background img 

Thank you for reviewing my PR

### CODE REVIEW

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Zainab-Alzubair , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost there


## Highlights

- [x] All linters are passing :heavy_check_mark: 
- [x] Good commit messages :heavy_check_mark: 
- [x] Neat and well organised code :heavy_check_mark: 

## Required Changes ♻️

- [ ]  Toolbar from figma design
Screenshot from 2022-09-22 21-14-58

 Your toolbar 
Screenshot from 2022-09-22 21-14-38

From the above images, I can see some disparities between your design and that of the figma. Please check and rectify that.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
[the remark this answers is not in the document]
Comment (Zainab-Alzubair) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Porfolio.
+
+
```
[the remark this answers is not in the document]
Comment (Zainab-Alzubair) [reply README.md]: Done

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <div class="nav-toggler"><div></div></div>
+    <div class="menu">
+        <div> 
+            <ul class="list-item">
+                <li><a href='#'>Portfolio</a></li>
+                <li><a href='#'>About</a></li>
+                <li><a href='#'>Contact</a></li>
+            </ul>
+        </div>
+    </div>
```
[the remark this answers is not in the document]
Comment (Zainab-Alzubair) [reply index.html]: Done

File: index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+             
+             <ul>
+              <li class="social"><a href="#"><img src="img\github.png"></a></li>
+              <li class="social"><a href="#"><img src="img\linkedin.png"></a></li>
+              <li class="social"><a href="#"><img src="img\Vector2.png"></a></li>
+              <li class="social"><a href="#"><img src="img\twitter.png"></a></li>
+              <li class="social"><a href="#"><img src="img\medium.png"></a></li>
+            </ul>
+      </div>
+      </section>
```
[the remark this answers is not in the document]
Comment (Zainab-Alzubair) [reply index.html]: Done

**Comment by Zainab-Alzubair** [comment]
Hi @jerryowusu 
Thank you for your feedback 

I've pushed the changes, kindly review them and let me know your insight 

Regards

File: README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+- GitHub: [@githubhandle](https://github.com/Zainab-Alzubair)
+- Twitter: [@twitterhandle](@zainabm34401029)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zainab-al-zubair-bb6777168/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (Nyame-Wolf) [inline README.md:40]: [OPTIONAL] Kindly add a link to the issues page of your project. Currently this opens an empty link

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @Zainab-Alzubair  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (Zainab-Alzubair) [reply README.md:40]: Will do so, thank you for your feedback @Nyame-Wolf

**Pull request merged**